test/e2e/specs/classic-editor.test.js

```diff
@@ -24,4 +24,42 @@ describe( 'classic editor', () => {
 		const textEditorContent = await page.$eval( '.wp-editor-area', ( element ) => element.value );
 		expect( textEditorContent ).toEqual( 'Typing in classic editor' );
 	} );
+
+	it( 'Should have wpautop disabled on a post containing blocks', async () => {
+		// Click the Text mode.
+		await page.click( '#content-html' );
+
+		// Enter some block text.
+		const original = `<!-- wp:paragraph -->
+<p>Foo bar four five six</p>
+<!-- /wp:paragraph -->
+
+This is another set of text`;
+		await page.type( '#content', original );
+
+		const initialTextEditorContent = await page.$eval( '.wp-editor-area', ( element ) => element.value );
+		expect( initialTextEditorContent ).toEqual( original );
```

Comment:
@tofumatt I made a little bit of progress with f93f875d4

At this point, the assertion is failing with:

<img width="571" alt="image" src="https://user-images.githubusercontent.com/36432/43268755-4691e0bc-90a6-11e8-9347-c373df263af6.png">

Reply:
I also tried:

```
await page.type( '#content', '<!-- wp:paragraph -->' );
await page.keyboard.press( 'Enter' );
await page.type( '#content', '<p>Foo bar four five six</p>' );
await page.keyboard.press( 'Enter' );
await page.type( '#content', '<!-- /wp:paragraph -->' );
await page.keyboard.press( 'Enter' );
await page.keyboard.press( 'Enter' );
await page.type( '#content', 'This is another set of text' );
```
But it fails with the same error.